src/csvcubed/utils/skos/codelist.py

```diff
@@ -80,18 +108,15 @@ def build_concepts_hierarchy_tree(
     tree = Tree()
     tree.create_node("root", identifier="root")
 
-    # Replacing empty values with None and sorting consepts by Sort Priortiy to maintain the order when iterating below.
+    # Replacing empty values with None.
```

Comment:
Do we not need to apply the sort order anymore?

Reply:
@robons I had a look at this. Sorting before constructing the tree wasn't actually needed in the first place. I believe I did this before the sort-order and sort-priority bug fixes and left it there. So this was always a redundant functionality.